backend/routes/events.js

```diff
@@ -71,7 +71,35 @@ router.get('/admin/all', async (req, res, next) => {
     } catch (err) {
         handleError(err, req, res, next);
     }
+});
+
+//Get all events  (non-admin)
+router.get('/non_admin/dashboard', async (req, res, next) => {
```

Comment:
to add to the previous comment:
router.get('/volunteer/dashboard', async (req, res, next)

Reply:
i'm going to flip your suggestion to /dashboard/volunteer because i think it'll be easier to distinguish but i like this